### ISSUE TITLE
Download button can show and be non-functional

### ISSUE THREAD

**Issue by emilyjablonski** [CityOfDetroit/bloom#1577]
[This is a listing on prod](https://homeconnect.detroitmi.gov/listing/6e11d372-edc8-4570-be57-7485884e3ac9/marwood_townhomes_67_77_marston_detroit_mi), which shows that we can currently get into a state where a listing has the Download Application button but it doesn’t do anything. This listing has Is there a paper application? as Yes and then they didn’t upload one. You can see the expected state where there is one uploaded on [this listing](https://homeconnect.detroitmi.gov/listing/7a087e1e-37b2-4e35-9168-a3d7a71b2a10/elmtree_apartments_2020_elmhurst_detroit_mi), it has a little language dropdown and the language is what you click to download. We either need to (1) continue to let people say “yes there is a paper app” and also not upload one and then just check for both on the public side, (2) require a paper app upload if you say there is one, (3) just remove the is there a paper app question and make the upload optional, (4) other suggestions

**Comment by eajensenwa** [comment]
@emilyjablonski @sarahlazarich suggestion #2 _require a paper app upload if you say there is one_ feels like the most appropriate behavior here from a UX perspective, followed by suggestion #3.

**Comment by sarahlazarich** [comment]
I agree that #2 feels like the best option, let's move forward with that.

**Comment by kathyccheng** [comment]
@sarahlazarich so sorry to add extra complexity here and sorry I missed this earlier, but there are some states to consider 

* If you say yes to paper applications, and don't upload a file, you can indicate that you can pick up an application or drop off an application using the application pick up or drop off fields in the address section. I think we do want to preserve the ability for people to indicate that. So I want to make sure that's still accounted for.

**Comment by sarahlazarich** [comment]
@kathyccheng , caught up with @eajensenwa on this -- we can associate the "download a paper application" button with the presence of a paper file. That way, the pick up / drop off sections will still show on the site, there just won't be a download button. Let me know how that sounds!

**Comment by sarahlazarich** [comment]
I think I'm seeing here that if you indicate that there is a paper application but don't upload one, you get an error and can't proceed with posting the listings, which is not the experience we were hoping for. We want property managers to be able to indicate that there is a paper app, and add in a drop off/pick up location and time.  

Also, reminder to please leave QA notes of tickets!

**Comment by sarahlazarich** [comment]
@KrissDrawing any thoughts on this?

**Comment by KrissDrawing** [comment]
@sarahlazarich strange, i can't edit / create any listing on Detroit dev site. It doesn't seem to be related to changes i made in this task 🤔 
I can see requested changes were applied for example [here](https://detroit-public-dev.netlify.app/listing/52616882-c06c-4230-8592-7f93f93e0880/655_hazelwood_655_hazelwood_st_detroit_mi). In partners we have `Is there a paper application?` set to `yes`, but no paper application uploaded and button is not displaying. I will follow that up on standup, as i ran out of ideas

**Comment by ludtkemorgan** [comment]
@sarahlazarich @KrissDrawing We had hit our db limit size in detroit dev which was preventing us from saving anything. I have deleted some old listings to bring us back under the limit so this is testable now. I'll also delete some more listings in the near future so we don't run into the limit again so soon

**Comment by sarahlazarich** [comment]
Ok, thank you, I'm seeing this change reflected now in Detroit! This looks good.
